src/pages/quiz/creation/QuestionsScreen.tsx

```diff
@@ -18,6 +17,8 @@ const getBaseUserAnswer = (q: Question): any => {
     switch (q.type) {
         case "multiple-choice":
             return Array(q.choices?.length || 0).fill(false);
+        case "single-choice":
```

Comment:
I haven't tested this PR change yet, but does it mean the first answer is already checked as soon as you enter the question?

Reply:
yup